probe_builder/builder/distro/flatcar.py

```diff
@@ -69,7 +69,7 @@ def build_kernel_impl(cls, config_hash, container_name, image_name, kernel_dir,
         else:
             logger.info("Build for {} probe {}-{} ({}) successful".format(label, coreos_kernel_release, config_hash, release))
 
-    def crawl(self, workspace, distro, crawler_distro, download_config=None):
+    def crawl(self, workspace, distro, crawler_distro, download_config=None, filter=''):
         kernels = crawl_kernels(crawler_distro)
```

Comment:
Shouldn't we pass the filter to `crawl_kernels`, too?
```suggestion
        kernels = crawl_kernels(crawler_distro, filter)
```

Reply:
the whole thing is going to be nuked anyway, but good point thanks!